refl1d/polymer.py

```diff
@@ -1,6 +1,4 @@
 # -*- coding: utf-8 -*-
-# This program is public domain
-# Authors Paul Kienzle, Richard Sheridan
```

Comment:
If effort was made to add attribution, I'd leave it in, as it doesn't hurt anything.  If we want to move this specific callout it to the license page or the documentation page that talks about attribution, that would be fine